firedrake/output/paraview_reordering.py

```diff
@@ -2,6 +2,14 @@
 import numpy as np
 from pyop2.utils import as_tuple
 
+try:
+    import vtkmodules.vtkCommonDataModel
+    del vtkmodules.vtkCommonDataModel
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "Error importing vtkmodules, you may need to install by executing\n\t"
+        "pip install vtk"
```

Comment:
Is this going to be robust enough? What is VTK's release schedule like?

Reply:
I've just checked the link you shared above. I'm convinced. They seem to be releasing wheels every month or so and are supporting the latest Pythons.